msticpy/auth/cloud_mappings.py

```diff
@@ -259,3 +276,13 @@ def suffix(self) -> azure_cloud.CloudSuffixes:
     def token_uri(self) -> str:
         """Return the resource manager token URI."""
         return f"{self.endpoints.resource_manager}.default"
+
+
+def return_defender_endpoint(cloud: str) -> str:
```

Comment:
maybe get_defender_endpoint?
Was thinking it might be better to use .get() rather than direct dict key access but probably better to raise an excpetion than take someone to an endpoint they weren't expecting